config/pipeline.yaml

```diff
@@ -1574,6 +1574,13 @@ jobs:
         - collabora-next:for-kernelci
     kcidb_test_suite: kselftest.devices-exist
 
+  kselftest-device-error-logs-main:
```

Comment:
[This also applies here](https://github.com/kernelci/kernelci-pipeline/pull/975#discussion_r1935330242) but it can be done as a next step.

I'd also like to see the diff of params applied to these job definitions (here and its ChromeOS counterpart) - are they that much different?